README.md

```diff
@@ -1,18 +1,25 @@
 # Hide LI (LinkedIn) Collab Articles
 
 ## Abstract
-This Chrome extension hides all collaborative articles on the LinkedIn homepage. 
+
+This Chrome extension hides all collaborative articles and AI generated prompts on the LinkedIn homepage.
 
 ## Tech Stack
+
 This project uses vanilla JS, CSS, and HTML.
 
-## Setup/Installation
+## Installation
+
 You can clone the repo locally using `git clone`.
 
-[This article](https://developer.chrome.com/docs/extensions/mv3/getstarted/development-basics/#load-unpacked) offers detailed instructions on loading an unpacked chrome extension. You can navigate to the extensions page via the puzzle piece icon in the toolbar. Enable "developer mode" on the top-right. From there, you can click "load unpacked extension" and choose the folder via the popup. You should then see the writing icon appear in the toolbar. When you click on it, you can see "Hide LI Collab Articles." From there, you can navigate to the LinkedIn homepage (linkedin.com/feed). As you scroll through your feed you shouldn't see any collaborative articles. If you do, please open an issue. 
+## Setup
+
+### Chrome
```

Comment:
I'm separating out the Chrome instructions in preparation for porting to Firefox in the future.